vortexasdk/version.py

```diff
@@ -1 +1 @@
-__version__ = "0.28.8"
+__version__ = "0.28.9"
```

Comment:
Does this actually warrant a version bump? Tests are not really part of the package clients get.